script/test/cmd/cmd_test.go

```diff
@@ -0,0 +1,775 @@
+	if err != nil {
+		fmt.Println("")
+		return "", "", errors.Wrap(err, "Error getting origin.")
+	}
+	origin = string(cmdOut)
+	origin = strings.TrimSpace(origin)
+	if !strings.Contains(origin, ".git") {
+		origin = origin + ".git"
+	}
+	brch, err := Git()
+	if err != nil {
+		log.Printf("%+v", err)
+		return
+	}
+	branch = strings.TrimSpace(brch)
+	fmt.Println("Test regarding build context running kompose from various directories")
+	fmt.Println("Buildconfig using " + origin + "::" + branch + " as source.")
+	return origin, branch, nil
+}
+
+// Kompose command for buildContext
+func KomposeCommand(command, provider, outFile, dockerComposeFile string) (error error) {
+	file := strings.Split(dockerComposeFile, "/")
+	cmdName := "kompose"
+	cmdArgs := []string{command, "--provider", provider, "-f", dockerComposeFile, "--stdout", "-j"}
+	out, err := exec.Command(cmdName, cmdArgs...).Output()
+	if err != nil {
+		return errors.Wrapf(err, "Error in %s application", file[1])
+	}
+	er := ioutil.WriteFile(outFile, out, 0644)
+	if er != nil {
+		return errors.Wrapf(err, "Error in writing file.")
+	}
+	return nil
+}
+
+// Kompose command
+func Kompose(command, provider, dockerComposeFile, result string) (cmd, output string, error error) {
+	b, error := ioutil.ReadFile(result)
+	if error != nil {
+		return "", "", errors.Wrap(error, "Error reading file")
+	}
+	file := strings.Split(dockerComposeFile, "/")
+	output = string(b)
+	cmdName := "kompose"
+	cmdArgs := []string{command, "--provider", provider, "-f", dockerComposeFile, "--stdout", "-j"}
+	out, err := exec.Command(cmdName, cmdArgs...).CombinedOutput()
+	if err != nil {
+		fmt.Println("\x1b[31;1m===> Test Fail <===\x1b[0m")
+		fmt.Println("Test Fail for", file[1])
+		return "", "", errors.Wrapf(err, "Error in %s application", file[1])
+	}
+	cmd = string(out)
+	fmt.Println("")
+	return cmd, output, nil
+}
+
+// Kompose command for Multiple files
+func MultipleFiles(command, provider, dockerComposeFileOne, dockerComposeFiletwo, result string) (cmd, output string, error error) {
+	b, error := ioutil.ReadFile(result)
+	if error != nil {
+		return "", "", errors.Wrap(error, "Error reading file")
+	}
+	fileOne := strings.Split(dockerComposeFileOne, "/")
+	filetwo := strings.Split(dockerComposeFiletwo, "/")
+	output = string(b)
+	cmdName := "kompose"
+	cmdArgs := []string{command, "--provider", provider, "-f", dockerComposeFileOne, "-f", dockerComposeFiletwo, "--stdout", "-j"}
+	out, err := exec.Command(cmdName, cmdArgs...).CombinedOutput()
+	if err != nil {
+		fmt.Println("\x1b[31;1m===> Test Fail <===\x1b[0m")
+		fmt.Println("Test Fail for", fileOne[1], filetwo[2])
+		return "", "", errors.Wrapf(err, "Error in %s application", fileOne[1], filetwo[2])
+	}
+	cmd = string(out)
+	fmt.Println("")
+	return cmd, output, nil
+}
+
+// Kompose command for bundle
+func KomposeBundle(command, dockerComposeFile, result string) (cmd, output string, error error) {
```

Comment:
Even for this, we should have one `Kompose` function to which we should be able to pass which kompose subcommand to run, and if we pass `bundle` there, then the `--bundle` flag should get substituted accordingly.
WDYT?